<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HornetWindow</class>
 <widget class="QWidget" name="HornetWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HornetWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="WaypointsLabel">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>10</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Waypoints</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="WaypointstList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>300</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>20</width>
     <height>721</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="AddWptButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>670</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Import from file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportCFButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>670</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Import from CF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportButtonToAC">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>670</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to aircraft</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModityWptButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveWptButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>680</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportButtonToFile">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>670</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to file</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>-10</y>
     <width>20</width>
     <height>671</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>655</y>
     <width>941</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="RAD1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>RAD 1 Presets</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RAD2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>RAD 2 Presets</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>31</width>
     <height>621</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>131</width>
     <height>612</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ChannelsLinesEdits">
    <item>
     <widget class="QLineEdit" name="line1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line5"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line6"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line7"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line8"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line9"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line10"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line11"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line12"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line13"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line14"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line15"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line16"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line17"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line18"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line19"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line20"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineM"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineG"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>71</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ChannelsLabels">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Channel 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Channel 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Channel 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Channel 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Channel 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Channel 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Channel 7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Channel 8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Channel 9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Channel 10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Channel 11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Channel 12</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Channel 13</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Channel 14</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Channel 15</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Channel 16</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Channel 17</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Channel 18</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Channel 19</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Channel 20</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Channel M</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Channel G</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>40</y>
     <width>71</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ChannelsLabels_2">
    <item>
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Channel 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Channel 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>Channel 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>Channel 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>Channel 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>Channel 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>Channel 7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>Channel 8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Channel 9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>Channel 10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>Channel 11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>Channel 12</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>Channel 13</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>Channel 14</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string>Channel 15</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string>Channel 16</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_42">
      <property name="text">
       <string>Channel 17</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_43">
      <property name="text">
       <string>Channel 18</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_44">
      <property name="text">
       <string>Channel 19</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_45">
      <property name="text">
       <string>Channel 20</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_46">
      <property name="text">
       <string>Channel M</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_47">
      <property name="text">
       <string>Channel G</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>131</width>
     <height>612</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ChannelsLinesEdits_2">
    <item>
     <widget class="QLineEdit" name="line1_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line2_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line3_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line4_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line5_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line6_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line7_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line8_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line9_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line10_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line11_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line12_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line13_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line14_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line15_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line16_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line17_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line18_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line19_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="line20_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineM_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineG_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="OptionsLabel">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>170</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>FUEL</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>BINGO</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Bingo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="maximum">
     <number>20000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>371</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>IFF</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>131</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode 1</string>
    </property>
    <widget class="QSpinBox" name="IFF_1_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="IFF_1_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>211</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Mode 3</string>
    </property>
    <widget class="QSpinBox" name="IFF_3_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="IFF_3_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="IFF_3_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="IFF_3_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
